bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/RPV_GluinoGluinoTo3jets/RPV_GluinoGluinoToJets_M-1000_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+set loop_optimized_output True
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model sm
```

Comment:
please follow these instructions (in particular please remove duplicate import comments, and lepton/neutrino definitions if they are not used) https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T